Missing package causing dapp confusion

This `develop` branch [requires the `form-data` module](https://github.com/OriginProtocol/origin-js/blob/develop/src/ipfs-service.js#L7). It is not included in the [`package.json`](https://github.com/OriginProtocol/origin-js/blob/develop/package.json). As a result, DApps get an exception when they import origin-js without having the module installed in their app.

**Here are the steps to reproduce FROM THE DAPP's PERSPECTIVE**:

_In a clean directory..._
```
git clone https://github.com/OriginProtocol/demo-dapp.git
cd demo-dapp
git checkout develop
npm install
npm start
```
`Module not found: Error: Can't resolve 'form-data' in '/Users/kgb/Dropbox/Origin/temp/demo-dapp/node_modules/origin/dist'`

People have been resorting to installing the `form-data` package in their DApp repos. I don't think this is the desired experience.

--- a/package.json
+++ b/package.json
@@ -32,8 +32,9 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "bs58": "^4.0.1",
-    "ipfs-api": "^14.3.7",
     "cross-fetch": "^2.1.1",
+    "form-data": "^2.3.2",
+    "ipfs-api": "^14.3.7",
     "map-cache": "^0.2.2",
     "npm": "^5.8.0",
     "truffle-contract": "^1.1.8",

